### ISSUE TITLE
Login Items on Mac does not show title Espanso, shows Federico Terzi instead

### ISSUE THREAD

**Issue by saravanabalagi** [espanso/espanso#1526]
**Describe the bug**
Login Items on MacOS shows Federico Terzi instead of Espanso. When cleaning up, people usually disable everything that they don't recognize and that could cause espanso to not start up.

**To Reproduce**
- Install Espanso on mac
- Check login items

**Expected behavior**
Expect Espanso in login items

**Screenshots**

<img width="481" alt="image" src="https://user-images.githubusercontent.com/8567893/218765598-70857eee-5ab3-4a2e-9c23-ffb0bad0b207.png">

**Setup information**
- OS: Mac OS 13.2 
- Version: 2.1.8

**Comment by smeech** [comment]
Is this still an issue with the current release, or can we close it?

**Comment by saravanabalagi** [comment]
@smeech I checked it again, but the same entry was still there after replacing the app. I assumed the entry didn't change because of the update, so removed the login entry manually, restarted the app and no new entry added. I removed the app, installed it fresh through brew, but it still did not create new login item.

**Comment by smeech** [comment]
Thank you - one of the Rust developers probably needs to make a small change to the code. It may already be in progress, because I know many instances are in the process of being changed.
@AucaCoyan?

**Comment by AucaCoyan** [comment]
Good morning! Thanks for re-checking. I'm sadly in base zero with mac, as I've never use one. but I'll keep an eye for the text and services around!

**Comment by smeech** [comment]
> Good morning! Thanks for re-checking. I'm sadly in base zero with mac, as I've never use one. but I'll keep an eye for the text and services around!

Just thought you're in closer contact with the development team. I had a look through the code but couldn't find it (lots of author, and URL mentions of Federico Terzi) but I wasn't sure I was able to find all of the 362 instances as I only had five pages of twenty.

**Comment by AucaCoyan** [comment]
Yes, we have to discuss how to transfer the ownership so that the app reflects the change `federico-terzi` - > `espanso`. This is still a thing that has not been started yet.

I investigate a little and I found that the repo has a bunch of scripts here and there. You have at least 2 forms to add an app to macOS Login, via `osascript` and `launchd` agent that uses `plist` files. We have a couple of those in the repo.
Checkout [this](https://serverfault.com/questions/299075/add-app-to-osx-login-items-from-bash-script) answer of serverfault.com.
